Honour byte order for ipv6prefix_lease env variable.

--- a/networking/udhcp/d6_dhcpc.c
+++ b/networking/udhcp/d6_dhcpc.c
@@ -333,7 +333,7 @@ static void option_to_env(const uint8_t *option, const uint8_t *option_end)
  * +-+-+-+-+-+-+-+-+
  */
 			move_from_unaligned32(v32, option + 4 + 4);
-			*new_env() = xasprintf("ipv6prefix_lease=%u", (unsigned)v32);
+			*new_env() = xasprintf("ipv6prefix_lease=%u", (unsigned)ntohl(v32));
 
 			sprint_nip6(ipv6str, option + 4 + 4 + 4 + 1);
 			*new_env() = xasprintf("ipv6prefix=%s/%u", ipv6str, (unsigned)(option[4 + 4 + 4]));

